src/main/java/com/odysseusinc/arachne/portal/service/ToPdfConverter.java

```diff
@@ -21,40 +21,71 @@
 package com.odysseusinc.arachne.portal.service;
 
 import com.odysseusinc.arachne.commons.utils.CommonFileUtils;
+
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
+import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import com.odysseusinc.arachne.portal.exception.ArachneSystemRuntimeException;
 import org.jodconverter.DocumentConverter;
 import org.jodconverter.document.DefaultDocumentFormatRegistry;
+import org.jodconverter.office.LocalOfficeManager;
 import org.jodconverter.office.OfficeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class ToPdfConverter {
 
-    public static final String docType = "docx";
+    public static final String DOC_TYPE = "docx";
+    private static final Logger log = LoggerFactory.getLogger(ToPdfConverter.class);
 
     @Autowired(required = false)
     private DocumentConverter converter;
 
+    @Autowired(required = false)
+    private LocalOfficeManager officeManager;
+
     public byte[] convert(final byte[] docFile) {
 
         if (converter == null) {
             return docFile;
         }
 
+        ensureJodConverterWorkingFolderExists();
+
         try (final ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
             converter
                     .convert(new ByteArrayInputStream(docFile))
                     .as(
-                            DefaultDocumentFormatRegistry.getFormatByExtension(docType))
+                            DefaultDocumentFormatRegistry.getFormatByExtension(DOC_TYPE))
                     .to(outputStream)
                     .as(DefaultDocumentFormatRegistry.getFormatByExtension(CommonFileUtils.TYPE_PDF))
                     .execute();
             return outputStream.toByteArray();
         } catch (final OfficeException | IOException e) {
-            throw new RuntimeException(e);
+            throw new ArachneSystemRuntimeException("Document conversion failure",e);
+        }
+    }
+
+    private void ensureJodConverterWorkingFolderExists() {
```

Comment:
This method's name does not reflect reality. This method creates directory in case it does not exist.